db/models.py

```diff
@@ -14,3 +14,41 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="actors")
```

Comment:
The `related_name` for the `actors` field in the `Movie` model is set to 'actors', which is not unique and can cause confusion. Consider using a more descriptive name like 'movies' to indicate the reverse relationship from Actor to Movie.